src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Diagnostics/RazorDiagnosticConverter.cs

```diff
@@ -24,7 +25,18 @@ public static Diagnostic Convert(RazorDiagnostic razorDiagnostic, SourceText sou
             throw new ArgumentNullException(nameof(sourceText));
         }
 
-        var diagnostic = new Diagnostic()
+        var projects = documentSnapshot is null
+            ? Array.Empty<VSDiagnosticProjectInformation>()
+            : [
+                new VSDiagnosticProjectInformation()
+                {
+                    Context = string.Empty,
```

Comment:
@davidwengier is there a reliable way we can get the name split between Name + TFM? I could parse but that seems wrong

Reply:
No, the only way we know it is from the slice dimensions from the project system. We can add it to the HostProject and plumb it through, just means we have to add it to project.razor.bin. how important is it?

Reply:
It just makes the display weird. If you look at the screenshot it's showing `Blazor1 (net6.0) ()` 

Might be able to work around

Reply:
yeah, thats pretty gross. Maybe the platform can fix it, and handle empty strings? Or what happens if `Context` is set to `null`?

Reply:
Looks like null worked out fine